project/profiles/capacity4more/modules/c4m/og/c4m_og_invite_users/c4m_og_invite_users.module

```diff
@@ -0,0 +1,100 @@
+<?php
+
+/**
+ * @file
+ * Invite existing users to a group or project.
+ */
+
+/**
+ * Implements hook_menu_alter().
+ */
+function c4m_og_invite_users_menu_alter(&$items) {
+  $items['group/%/%/admin/people/invite-users']['access callback'] = 'c4m_og_invite_users_access';
+  $items['group/%/%/admin/people/invite-users/%user']['access callback'] = 'c4m_og_invite_users_access';
+  $items['group/%/%/admin/people/invite-users/autocomplete']['access callback'] = 'c4m_og_invite_users_access';
+}
+
+/**
+ * Return true if the user is allowed to invite and false if he isn't.
+ *
+ * Checks by user role and og_role against group visibility (Public/Private) and
+ * Membership requests setting (Open/Moderated).
+ */
+function c4m_og_invite_users_access($group_type, $gid) {
+  if (_og_invite_access($group_type, $gid)) {
```

Comment:
I changed to return early. We should leave it above the others so that the permission can be configured through the OG permissions UI `(Invite any user)`.